.c3i/config_v2.yml

```diff
@@ -35,8 +35,9 @@ tasks:
     write_comments: false
     update_labels: false
     user_feedback:
-      title: "Conan v2 pipeline (informative, not required for merge)"
-      description: "> **Note**: Conan v2 builds are informative and they are not required for the PR to be merged."
+      title: "Conan v2 pipeline"
+      description: "> **Note**: Conan v2 builds may be required once they are on the [v2 ready](https://github.com/conan-io/conan-center-index/blob/master/.c3i/conan_v2_ready_references.yml) list"
```

Comment:
why `may be`? not always?

Reply:
There's a missing comma but I should be fine, the intention is clear about the condition